importer.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
+
+set -x
```

Comment:
Not necessary in production.

Reply:
I use it in production. it would trigger around 6 lines for every document, this is not much for 1000 documents which would get imported.